### PR TITLE
Dc eval revise

### PR DESCRIPTION
**Pull request by jakesabathia2**
Code format fixed and clean up.
Follow-up from https://github.com/apple/turicreate/pull/2932

### CODE REVIEW

File: src/toolkits/drawing_classifier/drawing_classifier.cpp [hunk 1]
```diff
@@ -51,6 +51,18 @@ struct result {
   data_iterator::batch data_info;
 };
 
+gl_sarray get_predictions_class(gl_sarray predictions_prob,
```
Comment (guihao-liang) [inline src/toolkits/drawing_classifier/drawing_classifier.cpp]: do you want to inline it? or make it a private/protected member function? Default static linkage is not recommended.

Comment (jakesabathia2) [reply src/toolkits/drawing_classifier/drawing_classifier.cpp]: Oh okay I'll make it private.

Comment (guihao-liang) [reply src/toolkits/drawing_classifier/drawing_classifier.cpp]: I think protected is better :-)

Comment (guihao-liang) [reply src/toolkits/drawing_classifier/drawing_classifier.cpp]: let's do protected inline. Just put the whole definition to the header file!

Comment (jakesabathia2) [reply src/toolkits/drawing_classifier/drawing_classifier.cpp]: good!

**Comment by jakesabathia2** [comment]
pass internal test.

**Pull request merged**